### DISCUSSION TITLE
Mounting a Large Volume to Frigate 0.14 LXC Container

### DISCUSSION THREAD

**Discussion by First777** [tteck/Proxmox#3535] [Help Needed]
I'm running Frigate 0.14 in an LXC container. By default, it creates a 20GB disk, which is insufficient for storing recordings and using it as a full-fledged NVR.
My question is: how do I mount a 4TB disk to the Frigate container? Since this isn't a Docker file, I can't specify the volume in the usual settings. Where would I configure this mounting point within the LXC environment?
<img width="716" alt="screenshot" src="https://github.com/user-attachments/assets/26058ee8-eb45-4eee-8a00-c2c534f7337b">
For reference, I've attached a screenshot of the folder I've added to the container (similar to "cctv_clips:frigate/media"). Please advise on which configuration file I should edit to achieve the desired mount.

**Comment by Nikhilse120** [comment]
Hi did you manage to change the  location of recording ? I have being looking for the same solution
